src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelReadingService.java

```diff
@@ -71,7 +71,7 @@ public XSSFWorkbook readWorkbook(String userToken, String documentBinaryUrl) thr
 
         } catch (IOException e) {
 
-            log.error("Error reading the Excel");
+            log.error("Error reading the Excel", e);
```

Comment:
I would delete this log.error completely as an exception is being thrown on the next line. You'll just get a duplicated error in the logs. Can some context be added to the exception message "Error reading the Excel" like case ref?

Reply:
deleted, added multiple reference